src/public/privatepage.php

```diff
@@ -7,13 +7,16 @@
     $dbPassword
 );
 
-$sql = 'SELECT * FROM pages';
+ $order = $_GET['order'];
```

Comment:
「どちらも選択されていない場合は、作成日の新しい順で表示」という要件があるので
以下のように修正しましょう

```suggestion
 $order = $_GET['order'] ?? 'desc';
```